src/Command/MigrateToJsonTypeCommand.php

```diff
@@ -22,6 +25,18 @@
  */
 class MigrateToJsonTypeCommand extends BaseCommand
 {
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    public function __construct(ManagerInterface $mediaManager, Pool $pool, EntityManagerInterface $entityManager)
```

Comment:
Whether this is an `EntityManagerInterface` or `Connection`, I guess it should be nullable since it may not exist.

Reply:
`Connection` will be more obvius. Done.